Mathlib/Analysis/NormedSpace/Multilinear/Basic.lean

```diff
@@ -1286,3 +1218,162 @@ instance continuousConstSMul : ContinuousConstSMul R (ContinuousMultilinearMap 
     (ContinuousLinearMap.compContinuousMultilinearMapL 𝕜 _ G G (c • ContinuousLinearMap.id 𝕜 G)).2⟩
 
 end SMul
+
+end Seminorm
+
+section Norm
+
+namespace ContinuousMultilinearMap
+
+/-! Results that are only true if the target space is a `NormedAddCommGroup` (and not just a
+`SeminormedAddCommGroup`).-/
+
+variable {𝕜 : Type u} {ι : Type v} {E : ι → Type wE} {G : Type wG} {G' : Type wG'} [Fintype ι]
+  [NontriviallyNormedField 𝕜] [∀ i, SeminormedAddCommGroup (E i)] [∀ i, NormedSpace 𝕜 (E i)]
+  [NormedAddCommGroup G] [NormedSpace 𝕜 G] [NormedAddCommGroup G'] [NormedSpace 𝕜 G']
+
+variable (f : ContinuousMultilinearMap 𝕜 E G)
+
+/-- A continuous linear map is zero iff its norm vanishes. -/
+theorem op_norm_zero_iff : ‖f‖ = 0 ↔ f = 0 := by
+  simp [← (op_norm_nonneg f).le_iff_eq, op_norm_le_iff f le_rfl, ext_iff]
+#align continuous_multilinear_map.op_norm_zero_iff ContinuousMultilinearMap.op_norm_zero_iff
+
+/-- Continuous multilinear maps themselves form a normed group with respect to
+    the operator norm. -/
+instance normedAddCommGroup : NormedAddCommGroup (ContinuousMultilinearMap 𝕜 E G) :=
+  NormedAddCommGroup.ofSeparation (fun f ↦ (op_norm_zero_iff f).mp)
+#align continuous_multilinear_map.normed_add_comm_group ContinuousMultilinearMap.normedAddCommGroup
+
+/-- An alias of `ContinuousMultilinearMap.normedAddCommGroup` with non-dependent types to help
+typeclass search. -/
+instance normedAddCommGroup' :
+    NormedAddCommGroup (ContinuousMultilinearMap 𝕜 (fun _ : ι => G) G') :=
```

Comment:
This instance doesn't need `G` to be Hausdorff.

Reply:
I made `G'` not necessarily Hausdordd and switched `G` and `G'`.